alecs-theme/src/WEB-INF/src/resources-importer/journal/templates/Member Card/Member Card.ftl

```diff
@@ -0,0 +1,21 @@
+<div class="card-row">
+  <#if Name.getSiblings()?has_content>
+    <#list Name.getSiblings() as cur_Name>
+      <div class="col-sm-12 col-md-6 col-lg-4">
+        <div class="card card-type-asset">
+	        <div class="aspect-ratio bg-checkered card-item-first">
+		        <img alt="Picture of ${cur_Name.getData()}"class="aspect-ratio-item-center-middle aspect-ratio-item-fluid" src="<#if cur_Name.Picture.getData()?? && cur_Name.Picture.getData() != "">${cur_Name.Picture.getData()}<#else>https://www.qualiscare.com/wp-content/uploads/2017/08/default-user.png</#if>">
```

Comment:
If we want a default image, we should add it to our theme's images folder and reference it there.

Reply:
Do not hardcode values as english.